packages/ui/src/fields/JSON/index.tsx

```diff
@@ -62,12 +63,44 @@ const JSONFieldComponent: JSONFieldClientComponent = (props) => {
       }
 
       const existingSchemas = monaco.languages.json.jsonDefaults.diagnosticsOptions.schemas || []
```

Comment:
@JarrodMFlesch since the existing schemas come from Monaco at runtime. And those are the schemas we need to make our comparison, this sanitization does need to occur at runtime.